crates/uv-configuration/src/target_triple.rs

```diff
@@ -610,6 +622,7 @@ impl TargetTriple {
             Self::Aarch64Manylinux238 => "",
             Self::Aarch64Manylinux239 => "",
             Self::Aarch64Manylinux240 => "",
+            Self::Wasm32Pyodide2024 => "3.1.58",
```

Comment:
Can you add a comment that this is the emscripten version, so future reviewers know what is emscripten and what is pyodide?

Reply:
Added.